tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -91,76 +84,89 @@ def check_read_nwb(self, nwbfile_path: str):
             video_type = Path(self.test_kwargs["file_paths"][0]).suffix[1:]
             assert f"Video: video_{video_type}" in nwbfile.acquisition
 
-    def check_align_starting_time_internal(self):
-        pass  # disabled since this interface follows a different strategy
+    def check_interface_align_timestamps(self):
+        all_unaligned_timestamps = self.interface.get_original_timestamps()
 
-    def check_align_starting_time(self):
-        fresh_interface = self.data_interface_cls(**self.test_kwargs)
+        generator = np.random.default_rng(seed=0)
```

Comment:
```suggestion
        rng = np.random.default_rng(seed=0)
```
This is the standard in numpy:
https://numpy.org/doc/stable/reference/random/generator.html

I would prefer the full random_number_generator but I def think it is better than generator.

Reply:
went with `random_number_generator`